### ISSUE TITLE
It is impossible to change position of products into a category in the catalog > product page (only the first time is OK)

### ISSUE THREAD

**Issue by khouloudbelguith** [PrestaShop/PrestaShop#14616]
**Describe the bug**
It is impossible to change the position of products into the catalog => product page.
After Drag & drop => Click on "Save & Refresh" button => The green alert "Products successfully sorted." is not displayed.
The new positions of products are not saved.
Only the first time is ok, but after some tests, this action is not possible
Checked with FireFox & Google Chrome.

Issue reported here: https://github.com/PrestaShop/PrestaShop/issues/11152#issuecomment-510818944

**To Reproduce**
1. Go to Catalog > Products
2. Click on Filter by categories and select a category ( Art category)
3. Click on Reorder button
4. Drag & drop some products
5. Click on "Save & Refresh" button
6. "Products successfully sorted." is not displayed & The new positions of products are not saved.

**Screenshots**
PS1.7.5.2: https://drive.google.com/file/d/1MD1EIi0yltRedcHbSEAL2ex2oC9Bb5ik/view
PS1.7.6.0: https://drive.google.com/file/d/1pbnjkFSTE7ebh6qX3Z2sEuZROAvkwEhL/view

**Additional information**
PrestaShop version: 1.7.5.2 / 1.7.6.0
PHP version: 7.0

**Comment by nsorosac** [comment]
Hi @khouloudbelguith 

Thank you for creating this issue. I am not sure its severity should be this low (minor) because its not an isolated use-case: every merchant is concerned by this bug, and the reordering "feature" is very important for the merchants.

All our prestashop customers are impacted, and they are all hoping we can do something for them quickly :-(

@mickaelandrieu this is the products BO reordering issue we mentioned last week during your training session :-)

**Comment by nsorosac** [comment]
Hi @marionf ! Here is a glimpse of customer feedback on this bug: 
![bug-PS17-reorder](https://user-images.githubusercontent.com/54878/61959564-f8d98800-afc3-11e9-83ed-cd9c7b857d51.png)

**Comment by PierreRambaud** [comment]
It's a POC but you can tell me if it's working for you too https://github.com/PrestaShop/PrestaShop/pull/14879

**Comment by nsorosac** [comment]
Thanks @PierreRambaud we will try this on several projects !

**Comment by PierreRambaud** [comment]
@nsorosac Just one to make sure it works 😅

**Comment by sarahdib** [comment]
The problem is fixed if we use the icon in position column. The product is selected and we can change the position and saved and all changes is validated.

Since this PR correct the problem it's OK a New issue was created for the second issue : #15291

**Comment by loetje88** [comment]
Running on PrestaShop-versie: 1.7.6.4 , problem still persists.

Almost every button in back office no product page does not work by clicking, only after refresh.
Sorting backoffice sorting does not works FO.
